docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -18,56 +18,40 @@ different labels.
 The following example will set up the job with certain attributes (scrape
 intervals, query parameters) and let it scrape two instances of the blackbox
 exporter. The received metrics will be sent over to the provided list of
-remote_writes, as defined by other components.
+remote writes, as defined by other components.
```

Comment:
nit: receivers instead of "remote writes?" Elements in forward_to aren't always prometheus.remote_write, sometimes it's prometheus.relabel, and the list of things it could be will probably grow over time.